{"uuid":"666175af-a8bd-40b6-97f0-a51296c72233","historyId":"3dabc613d3726498ae543ae3963774d4","testCaseId":"[engine:junit-jupiter]/[class:br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT]/[nested-class:ObterClientes]/[method:obterTodos()]","testCaseName":"obterTodos()","fullName":"br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT$ObterClientes.obterTodos","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT]/[nested-class:ObterClientes]/[method:obterTodos()]"},{"name":"host","value":"DESKTOP-O4LPNQJ"},{"name":"thread","value":"14260@DESKTOP-O4LPNQJ.Test worker(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT$ObterClientes"},{"name":"testClass","value":"br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT$ObterClientes"},{"name":"testMethod","value":"obterTodos"},{"name":"suite","value":"br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT.ObterClientes"}],"links":[],"name":"obterTodos()","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not open JPA EntityManager for transaction","trace":"org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction\r\n\tat org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:466)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:531)\r\n\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:405)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:617)\r\n\tat org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:386)\r\n\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\tat org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:137)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\tat org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:164)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)\r\n\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\tat org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:223)\r\n\tat jdk.proxy3/jdk.proxy3.$Proxy133.saveAll(Unknown Source)\r\n\tat br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT.setup(ClienteControllerIT.java:46)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tSuppressed: org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction\r\n\t\tat org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:466)\r\n\t\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:531)\r\n\t\tat org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:405)\r\n\t\tat org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:617)\r\n\t\tat org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:386)\r\n\t\tat org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)\r\n\t\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\t\tat org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:137)\r\n\t\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\t\tat org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:164)\r\n\t\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\t\tat org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)\r\n\t\tat org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184)\r\n\t\tat org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:223)\r\n\t\tat jdk.proxy3/jdk.proxy3.$Proxy133.deleteAll(Unknown Source)\r\n\t\tat br.com.cadastrocliente.mscadastrocliente.application.controller.ClienteControllerIT.dropDatabase(ClienteControllerIT.java:166)\r\n\t\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\t\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\t\t... 3 more\r\n\tCaused by: org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection [Connection to localhost:8081 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.] [n/a]\r\n\t\tat org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:100)\r\n\t\tat org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:58)\r\n\t\tat org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)\r\n\t\tat org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)\r\n\t\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.acquireConnectionIfNeeded(LogicalConnectionManagedImpl.java:116)\r\n\t\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.getPhysicalConnection(LogicalConnectionManagedImpl.java:143)\r\n\t\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.getConnectionForTransactionManagement(LogicalConnectionManagedImpl.java:273)\r\n\t\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.begin(LogicalConnectionManagedImpl.java:281)\r\n\t\tat org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.begin(JdbcResourceLocalTransactionCoordinatorImpl.java:232)\r\n\t\tat org.hibernate.engine.transaction.internal.TransactionImpl.begin(TransactionImpl.java:83)\r\n\t\tat org.springframework.orm.jpa.vendor.HibernateJpaDialect.beginTransaction(HibernateJpaDialect.java:176)\r\n\t\tat org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:420)\r\n\t\t... 20 more\r\n\tCaused by: org.postgresql.util.PSQLException: Connection to localhost:8081 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.\r\n\t\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:342)\r\n\t\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)\r\n\t\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:263)\r\n\t\tat org.postgresql.Driver.makeConnection(Driver.java:443)\r\n\t\tat org.postgresql.Driver.connect(Driver.java:297)\r\n\t\tat com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)\r\n\t\tat com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)\r\n\t\tat com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)\r\n\t\tat com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)\r\n\t\tat com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)\r\n\t\tat com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:100)\r\n\t\tat com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)\r\n\t\tat org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:122)\r\n\t\tat org.hibernate.internal.NonContextualJdbcConnectionAccess.obtainConnection(NonContextualJdbcConnectionAccess.java:46)\r\n\t\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.acquireConnectionIfNeeded(LogicalConnectionManagedImpl.java:113)\r\n\t\t... 27 more\r\n\tCaused by: java.net.ConnectException: Connection refused: no further information\r\n\t\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\r\n\t\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)\r\n\t\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)\r\n\t\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\r\n\t\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\r\n\t\tat java.base/java.net.Socket.connect(Socket.java:633)\r\n\t\tat org.postgresql.core.PGStream.createSocket(PGStream.java:243)\r\n\t\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\r\n\t\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:132)\r\n\t\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:258)\r\n\t\t... 41 more\r\nCaused by: org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC Connection [Connection to localhost:8081 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.] [n/a]\r\n\tat org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:100)\r\n\tat org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:58)\r\n\tat org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)\r\n\tat org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)\r\n\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.acquireConnectionIfNeeded(LogicalConnectionManagedImpl.java:116)\r\n\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.getPhysicalConnection(LogicalConnectionManagedImpl.java:143)\r\n\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.getConnectionForTransactionManagement(LogicalConnectionManagedImpl.java:273)\r\n\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.begin(LogicalConnectionManagedImpl.java:281)\r\n\tat org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.begin(JdbcResourceLocalTransactionCoordinatorImpl.java:232)\r\n\tat org.hibernate.engine.transaction.internal.TransactionImpl.begin(TransactionImpl.java:83)\r\n\tat org.springframework.orm.jpa.vendor.HibernateJpaDialect.beginTransaction(HibernateJpaDialect.java:176)\r\n\tat org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:420)\r\n\t... 19 more\r\nCaused by: org.postgresql.util.PSQLException: Connection to localhost:8081 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:342)\r\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)\r\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:263)\r\n\tat org.postgresql.Driver.makeConnection(Driver.java:443)\r\n\tat org.postgresql.Driver.connect(Driver.java:297)\r\n\tat com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)\r\n\tat com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)\r\n\tat com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)\r\n\tat com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)\r\n\tat com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)\r\n\tat com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:100)\r\n\tat com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)\r\n\tat org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:122)\r\n\tat org.hibernate.internal.NonContextualJdbcConnectionAccess.obtainConnection(NonContextualJdbcConnectionAccess.java:46)\r\n\tat org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.acquireConnectionIfNeeded(LogicalConnectionManagedImpl.java:113)\r\n\t... 26 more\r\nCaused by: java.net.ConnectException: Connection refused: no further information\r\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\r\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\r\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\r\n\tat java.base/java.net.Socket.connect(Socket.java:633)\r\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:243)\r\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:132)\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:258)\r\n\t... 40 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1715997106191,"stop":1715997108231}